tests/input/examples/schema_definition-native-array-rgb.yaml

```diff
@@ -0,0 +1,44 @@
+id: https://example.org/arrays
+name: arrays-temperature-example
+title: Array Temperature Example
+description: |-
+  Example LinkML schema to demonstrate a 3D DataArray of temperature values with labeled axes
+license: MIT
+
+prefixes:
+  linkml: https://w3id.org/linkml/
+  wgs84: http://www.w3.org/2003/01/geo/wgs84_pos#
+  example: https://example.org/
+
+default_prefix: example
+
+imports:
+  - linkml:types
+
+classes:
+
+  RGBImage:
+    attributes:
+      rgb:
+        range: float
```

Comment:
```suggestion
        range: float
        multivalued: true
```

Is this necessary?

Reply:
multivalued will be implicit based on the presence of `array_info`. There are a few other cases like that e.g. `inlined_as_dict -> inlined`